### DISCUSSION TITLE
Search for keywords in the player

### DISCUSSION THREAD

**Discussion by cmprmsd** [asciinema/discussions#162] [Ideas]
# Request
It would be awesome to be able to search through one or many recordings for certain keywords and jump to that video at the correct timestamp where it appears. 

e.g. I use a tool with a certain keyword `nmap hostname` and want to search for this hostname in all of my terminal logs to find the occurrences and the commands output.

# Technical Background
I'm currently running https://github.com/H8to/cLoggy in order to record my sessions and output them into a log folder all together. 
From a technical perspective it looks like this:
```
logfile=$logdir/tmux_$(date +%F_%T)_PID$$.html
tmux pipe-pane "exec cat - | ansifilter --html --font monospace >> $logfile"
```
However, tmux pipe-panes and the html output do not work that well for interactive applications like less, vim etc. which is why I'd like to migrate to this awesome project and have my logs asciinemized :D 

Edit:
I just found out there is no "startAt" parameter for the command CLI yet. But there is your timestamp in each file so a search could be easily implemented externally if there was a startAt parameter that accepts the internal timecode (seconds.fraction)

**Comment by bkil** [comment]
Related: https://github.com/asciinema/asciinema-server/issues/111
